man/CMakeLists.txt

```diff
@@ -1,43 +1,60 @@
-install(FILES
-	ceph-osd.8
-	ceph-mds.8
-	ceph-mon.8
-	ceph-syn.8
-	crushtool.8
-	osdmaptool.8
-	monmaptool.8
-	ceph-conf.8
-	ceph-run.8
-	ceph.8
-	mount.ceph.8
-	ceph-create-keys.8
-	radosgw.8
-	radosgw-admin.8
-	ceph-authtool.8
-	rados.8
-	librados-config.8
-	ceph-clsinfo.8
-	ceph-debugpack.8
-	cephfs.8
-	ceph-dencoder.8
-	ceph-rest-api.8
-	ceph-post-file.8
-  DESTINATION ${CEPH_MAN_DIR}/man8)
+set(manpages_srcs
```

Comment:
we list sphinx-build as one of the build dependencies, so i put the generated man pages into the build directory instead of in "/man".

also it might be more natural if we have a CMakeLists.txt in into "/doc/man/8", and collect the man pages there. but i am good either way.